src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+---
+layout: article
+title: Team invites
+description: Learn how to manage team invites in Appwrite. Implement both client-side email invites and server-side custom flows for team memberships.
+---
+
+Appwrite provides two approaches for adding members to teams: client-side email invites and server-side custom flows. Each approach serves different use cases and offers unique benefits.
+
+# Client-side email invites {% #client-side %}
+
+Client-side email invites provide a user-friendly way to add members to your team. When creating a membership, Appwrite:
+1. Sends an automated email invitation to the user
+2. Creates a pending membership
+3. Activates the membership when the user accepts
+
+Client-side invites are ideal when you want a simple, automated process.
+Appwrite handles the email delivery with built-in templates and localization support, making it easy to implement a standard invite acceptance flow with email verification.
+
+{% multicode %}
+```client-web
+import { Client, Teams } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const teams = new Teams(client);
+
+// Create membership with email invite
+const membership = await teams.createMembership(
+    'team_id',
+    ['developer'],     // roles
+    'user@example.com' // email
+);
+```
+```client-flutter
+import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+final teams = Teams(client);
+
+// Create membership with email invite
+final membership = await teams.createMembership(
+    teamId: 'team_id',
+    roles: ['developer'],
+    email: 'user@example.com'
+);
+```
+```client-apple
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+
+let teams = Teams(client)
+
+// Create membership with email invite
+let membership = try await teams.createMembership(
+    teamId: "team_id",
+    roles: ["developer"],
+    email: "user@example.com"
+)
+```
+```kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+
+val teams = Teams(client)
+
+// Create membership with email invite
+val response = teams.createMembership(
+    teamId = "team_id",
+    roles = listOf("developer"),
+    email = "user@example.com"
+)
+```
+{% /multicode %}
+
```

Comment:
accepting the invite should go here since it's only applicable to client-side invites.